docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -33,3 +66,27 @@ There is no a universal recipe for determining the mean/scale values for a parti
 ## When to Reverse Input Channels <a name="when_to_reverse_input_channels"></a>
 Input data for your application can be of RGB or BRG color input order. For example, OpenVINO Samples load input images in the BGR channels order. However, the model may be trained on images loaded with the opposite order (for example, most TensorFlow\* models are trained with images in RGB order). In this case, inference results using the OpenVINO samples may be incorrect. The solution is to provide `--reverse_input_channels` command line parameter. Taking this parameter, the Model Optimizer performs first convolution or other channel dependent operation weights modification so these operations output will be like the image is passed with RGB channels order.
 
+## Compression of model to FP16
+
+Mode Optimizer can compress the model to `FP16` data type. This will not only make the
+model to occupy less space in file system, but also may increase performance on some
+hardware. Model Optimizer will change data type on all the constants inside the model
+to `FP16` precision and insert `Convert` nodes to initial data type so that the data
+flow inside the model is preserved. To compress the model to `FP16` please use
+`--data_type` option like this:
+
+```
+mo --input_model /path/to/model --data_type FP16
+```
+
+> **NOTE**: Using `--data_type FP32` will not do anything and will not force `FP32` 
+> precision in the model. If the model was `FP16` originally in the framework,
+> Model Optimizer will not convert such weights to `FP32` even if `--data_type FP32`
+> option is used .
+
+Some plugins, for example GPU, will show greater performance while slightly sacrificing
```

Comment:
greater => better?
@vladimir-paramuzov do we need to write about GPU here? Maybe we just need to refer to GPU paragraph with precisions description?

Reply:
Although they are often synonymous, we can change "greater" to "higher". But more importantly, I agree, this line is not necessary, as it is mentioned more or less in the beginning paragraph. We could just mention GPU there. And the following line should also go to the top, as it is another reason for compressing, just like the size and performance boost.